documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    proc(arg1, arg2, *args)       |    apply(proc, arg1, arg2, args)              |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    map(proc, list1, list2, ...)  |    map(proc, list1, list2, ...)               |
+|    # returns iterator in         |                                               |
+|    # Python 3                    |                                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    map(proc, vec1, vec2, ...)    |    map(proc, vec1, vec2, ...)                 |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    map(proc, str1, str2)         |    map(proc, str1, str2)                      |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    for a,b in zip(list1, list2): |    do(proc, list1, list2)                     |
+|      proc(a, b)                  |                                               |
++----------------------------------+-----------------------------------------------+
+
+Continuations
+-------------
+
+Python doesn’t have first-class continuations like Scheme’s
+``call/cc``, and Dylan’s ``block`` syntax is more like structured
+exception handling or generators.
+
+Here are approximate equivalents:
+
++-----------------------------------+-----------------------------------------------+
+| Python                            | Dylan                                         |
++===================================+===============================================+
+| .. code-block:: python            | .. code-block:: dylan                         |
+|                                   |                                               |
+|    def example():                 |    block (k)                                  |
+|      def k():                     |      body                                     |
+|        pass                       |    end block                                  |
+|      body(k)                      |                                               |
```

Comment:
Not sure what you want to express here.  ?

Reply:
Is "Lost in Python translation". What is the equivalent of the `block` in Opendylan in Python?

Reply:
There's nothing directly equivalent to `block`, but Pythonistas will be interested in how to do `try/except/finally`, which has direct equivalents in `block`.

You could add [a non-local exit example](https://play.opendylan.org/shared/a32e7c1a359f8753) and put `N/A` on the Python side.